python/lsst/ip/isr/binImageDataTask.py

```diff
@@ -0,0 +1,214 @@
+# This file is part of ip_isr.
```

Comment:
I think github is telling me this was a `git mv` rename of `binExposureTask.py`.  Is that correct (I'm hoping it's correct).

Reply:
Yes - that's correct. I split the edits and the git mv into two separate commits, so you can see only the edits against the current main.

I thought it would be pretty cruel make the edits and the mv in the same commit!!